include/boost/gil/image_processing/fast_feature_detector.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef BOOST_GIL_IMAGE_PROCESSING_FAST_HPP
+#define BOOST_GIL_IMAGE_PROCESSING_FAST_HPP
+
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/locator.hpp>
+#include <boost/gil/point.hpp>
+#include <algorithm>
+#include <cmath>
+#include <vector>
+namespace boost { namespace gil {
+namespace detail {
+template <typename srcview>
+bool fast_feature_detector(const srcview& buffer, int r, int c, std::vector<point_t>& points, int t)
```

Comment:
I recommend "eastconst": `SrcView const& buffer`
https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/

Reply:
Done.